app/views/gallery/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <%= title(t('gallery.title')) %>
 <div class="headline headline-md">
-  <h1><%= t('gallery.title') %></h1>
+  <h1>
+    <%= t('gallery.title') %> -
+    <% if @year.present? %>
```

Comment:
Right now we are:
- Showing albums for one year at a time
- Showing a list of other years with albums

I want to show which year is showing, I would want `params[:year] = nil` to default to current year.
I do not want the current year to be in the list of years.

Is there a smooth way to do this without two instance variables?